sambacc/ctdb.py

```diff
@@ -181,6 +181,29 @@ def read_ctdb_nodes(path: str = CTDB_NODES) -> list[str]:
     return entries
 
 
+class PublicAddrAssignment(typing.TypedDict):
+    address: str
+    interfaces: list[str]
+
+
+def _ensure_public_addresses_file(
+    path: str, addrs: list[PublicAddrAssignment]
+) -> None:
+    with open(path, "w") as fh:
+        _write_public_addresses_file(fh, addrs)
+
+
+def _write_public_addresses_file(
+    fh: typing.IO, addrs: list[PublicAddrAssignment]
+) -> None:
+    for entry in addrs:
+        fh.write(f"{entry['address']}")
```

Comment:
Minor issue. Consider it if you need to update this PR for other reasons.

```suggestion
        fh.write(entry['address'])
```